app/views/govuk_publishing_components/components/_related_navigation.html.erb

```diff
@@ -0,0 +1,76 @@
+<% related_nav_helper = RelatedNavigationHelper.new %>
+<% related_content = related_nav_helper.build_related_content(local_assigns) %>
+<% if related_content.map(&:values).flatten.any? || related_nav_helper.other.flatten.any? %>
+<% random = Random.new.rand(100) %>
```

Comment:
This does seem hacky, and I noticed it doesn't always work (I guess because the same random number is sometimes generated more than once on the same page). Perhaps this shouldn't block this PR, but a separate issue should be raised around looking into if the ID errors can be ignored in component guide pages?

Reply:
Yes definitely an issue to add to the backlog, as it's not an ideal solution.

Reply:
Consider using `SecureRandom` which should guarantee randomness 

See https://github.com/alphagov/govuk_publishing_components/blob/a323d988e691d4e44c8e91db3ad92ab33d869a63/app/views/govuk_publishing_components/components/_input.html.erb#L2

Reply:
Good shout, thanks @nickcolley